ui/tests/acceptance/secrets/backend/ssh/role-test.js

```diff
@@ -44,12 +44,10 @@ module('Acceptance | secrets/ssh', function(hooks) {
   });
 
   test('it deletes a role', async function(assert) {
-    await mountAndNav(assert);
+    const path = await mountAndNav(assert);
     await editPage.createOTPRole('role');
-    await showPage.edit();
-    assert.equal(currentRouteName(), 'vault.cluster.secrets.backend.edit', 'navs to the edit page');
-
-    await editPage.deleteRole();
+    await showPage.visit({ backend: path, id: 'role' });
```

Comment:
I _think_ we should end up on the show page already here (but it probably doesn't hurt to explicitly nav there).